src/core/public/chrome/ui/header/recent_items.tsx

```diff
@@ -73,7 +73,12 @@ const bulkGetDetail = (savedObjects: Array<Pick<SavedObject, 'type' | 'id'>>, ht
         .get<SavedObjectWithMetadata>(
           `/api/opensearch-dashboards/management/saved_objects/${encodeURIComponent(
             obj.type
-          )}/${encodeURIComponent(obj.id)}`
+          )}/${encodeURIComponent(obj.id)}`,
```

Comment:
Should it filter out objects that's not belong to the current workspace before bulkGetDetails?

Reply:
According to previous decision / discussion, recent assets is a global concept so no matter in which workspace we will still show recent assets from other workspace.